### PR TITLE
oh-my-posh@7.85.4: update env_set and others

### PR DESCRIPTION
**Pull request by ClassicDarkChocolate**
- notes: Link is outdated.
- env_set: Currently `Get-PoshThemes` is not working because `$env:POSH_THEMES_PATH` is not correctly set.
- persist: Allow themes to get updated.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/oh-my-posh.json [hunk 1]
```diff
@@ -3,15 +3,20 @@
     "description": "A prompt theme engine for any shell",
     "homepage": "https://ohmyposh.dev",
     "license": "MIT",
-    "notes": "Refer to 'https://ohmyposh.dev/docs/windows#replace-your-existing-prompt' for shell specific configurations.",
+    "notes": [
+        "Thanks for installing Oh My Posh.",
+        "Have a look at https://ohmyposh.dev for detailed instructions for your shell."
```
Comment (tech189) [inline bucket/oh-my-posh.json:8]:
```suggestion
        "Have a look at https://ohmyposh.dev/docs/installation/prompt for detailed instructions for your shell."
```

**Review by tech189 — COMMENTED** [review]
Added a link direct to the relevant docs for adding it to your shell

File: bucket/oh-my-posh.json (hunk 1 above)
Comment (ClassicDarkChocolate) [reply bucket/oh-my-posh.json]: Thanks!

File: bucket/oh-my-posh.json [hunk 2]
```diff
@@ -3,15 +3,20 @@
     "description": "A prompt theme engine for any shell",
     "homepage": "https://ohmyposh.dev",
     "license": "MIT",
-    "notes": "Refer to 'https://ohmyposh.dev/docs/windows#replace-your-existing-prompt' for shell specific configurations.",
+    "notes": [
+        "Thanks for installing Oh My Posh.",
+        "Have a look at https://ohmyposh.dev/docs/installation/prompt for detailed instructions for your shell."
+    ],
```
Comment (rashil2000) [inline bucket/oh-my-posh.json:9]:
```suggestion
    "notes": "Refer to 'https://ohmyposh.dev/docs/installation/prompt' for shell specific configurations.",
```

Comment (ClassicDarkChocolate) [reply bucket/oh-my-posh.json:9]: I originally copied the notes from the dev. But it is okay for simplicity.

**Pull request merged**